paths/reset/fc-commitbroke.md

```diff
@@ -0,0 +1,71 @@
+title: Commit Broke Everything!
+permalink: /reset/fc-06
+next-page: /reset/fc-01
+facilitator: false
+sidebar:
+  nav: "advanced"
+main-content: |  
+
+  Your well intentioned commit was supposed to fix that bug that has been plaguing your project for weeks, but after making the commit things are worse than they were before! First, breathe, you can fix this; for real. Second, remember you aren't the first person who completely broke everything with a commit, heck, even the GitHub Trainers do it from time to time. So, now that you have found yourself in some :ahem: _very_ distinguished company, you can fix that gross commit.  
+
+pushed: |
+    Pushing your commit isn't the end of the world, other than it breaking everything, but still, the world will be fine. So, since the commit has been pushed and other collaborators might have begun working on the files related to that commit, especially if they noticed that the commit broke even more things.
+    You can fix it really quickly though, using a handful of commands and soon it everything will be back to normal.
+
+     1. Ensure you are on the correct branch and enter: `git log --oneline`.
+     1. Identify the SHA-1 hash for the **adding file 2** commit.
+     1. Enter: `git revert SHA-1`, where the SHA-1 is the SHA-1 for the **adding file 2** commit. You will be prompted to edit a commit message for the revert, however, you don't need to change the supplied message if you don't want to. Once you are happy with the commit message, close your editor.
+     1. Enter: `git log --oneline`. You know have a commit for the revert that you can push up to your remote.
+     1. Enter: `git push`.
+
+    For some extra practice using `git revert`, try to `revert` your previous `revert` and `push`ing that back up to your remote.
+
+didnt-push: |
+    Well, you didn't push the commit, that means no one else knows you made the project worse than it was. Use the following steps to fix that errant commit.
+
+    Use Git Reset to bring the file back to the working directory.
+
+    Make sure to use `ls` in this workflow as we reference it in the next sub-section
+
+    ## I Just Want it All Gone
+    Sometimes the best way to fix a problem is to pretend it never existed in the first place. If after removing the commit you just don't have the patience to deal with this garbage commit, you can use Git to make it look like it never happened. In the previous steps, you brought the file back to the Working Directory, so you could fix the issues you introduced with the initial commit and create a brand new commit that fixed that nagging bug and removed any trace of your previous attempt to fix it. If trying to identify how to get your fix to work is just not getting anywhere, we can use a different command to just get rid of it.
+
+    > If you want to see something kinda cool, open your local repository in a file browser (Finder, My Computer, etc.) and leave it to the side (but in view).
```

Comment:
😍 This formatting looks really cool. It stands out but doesn't make it too busy.